src/main/java/net/sf/jabref/gui/groups/GroupSelector.java

```diff
@@ -702,6 +712,11 @@ public void componentOpening() {
     }
 
     @Override
+    public String getSidePaneName() {
```

Comment:
I need to access the name from the `SidePaneComponent`.
Sadly you cannot define an abstract static method